CHANGELOG.unreleased.md

```diff
@@ -23,6 +23,7 @@
 - Fix default filters not being set in all tables (PR [#4994](https://github.com/vatesfr/xen-orchestra/pull/4994))
 - [SDN Controller] Broken encrypted tunnels after host reboot [#4996](https://github.com/vatesfr/xen-orchestra/pull/4996)
 - Don't log server's credentials in case of `SESSION_AUTHENTICATION_FAILED` error (PR [#4995](https://github.com/vatesfr/xen-orchestra/pull/4995))
+- [Plugin/backup-reports] Fix `No recipients defined` error (PR [#4998](https://github.com/vatesfr/xen-orchestra/pull/4998))
```

Comment:
Unclear.

Reply:
`- [Plugin/backup-reports] Fix `No recipients defined` error in case of an empty backup recipients` ? IMHO, saying that this error was fixed is enough.

Reply:
I disagree because it's unclear when the error occurs.

Maybe something like?

```suggestion
- [Plugin/backup-reports] Always use plugin addresses when none defined in backup job (PR [#4998](https://github.com/vatesfr/xen-orchestra/pull/4998))
```

Reply:
IMHO, this message describes what the PR do, not what the user see. 

The user only saw the error `No recipients defined` triggered by the plugin, he will not know if this entry will fix it.

Reply:
```suggestion
- [Plugin/backup-reports] Fix `No recipients defined` error when recipients defined at plugin level (PR [#4998](https://github.com/vatesfr/xen-orchestra/pull/4998))
```

Reply:
`when recipients are defined at the plugin level` or `when the report recipients are defined at the plugin level` ?

Reply:
Why do you not change it to `when the report recipients are defined at the plugin level` ?

Reply:
I did this suggestion because i thought that `are` is needed.